TF_Service_dotNet/TouchFree/Interactions/InteractionModules/AirPushInteraction.cs

```diff
@@ -277,14 +272,14 @@ private float GetAppliedForceChange(float _currentVelocity, float _dt, Vector2 _
                     // Adjust force based on spring stiffness
 
                     // Perform a calculation:
-                    float vClamped = Mathf.Clamp(Mathf.Abs(_currentVelocity), speedMin, speedMax);
+                    float vClamped = Math.Clamp(Math.Abs(_currentVelocity), speedMin, speedMax);
 
                     float ratio = (vClamped - speedMin) / (speedMax - speedMin);
```

Comment:
Can this be more descriptive now that it isn't explicitly used against a 'stiffness curve'? Perhaps stiffnessRatio?